crates/bevy_ecs/src/component.rs

```diff
@@ -378,6 +380,7 @@ impl ComponentDescriptor {
             storage_type,
             is_send_and_sync: true,
             type_id: None,
+            is_resource: false,
```

Comment:
Unlike `new`, there's no requirement that this method only be used for component types.

Reply:
Yeah I can add an argument for that. Seems like every one of my PRs is bound to have at least one breaking change at this point 😄